Sources/XcodeGenKit/CarthageDependencyResolver.swift

```diff
@@ -159,6 +159,8 @@ extension Platform {
             return "watchOS"
         case .macOS:
             return "Mac"
+        case .visionOS:
+            fatalError("Not supported.")
```

Comment:
Please check `GeneratedPerformanceTests.testGeneration`. It looks this test refers this `fatalError` case and that's why test is failed. Please run test on local.

Reply:
Hints:

- ↓ CI test is failed due to `fatalError` of mentioned line

https://github.com/yonaskolb/XcodeGen/blob/49a9bda9925b4df5b87db912d84c10c418912bb8/Tests/PerformanceTests/PerformanceTests.swift#L24-L30

- And, as you can see, this test is using `testProject` function which defined here

https://github.com/yonaskolb/XcodeGen/blob/49a9bda9925b4df5b87db912d84c10c418912bb8/Tests/PerformanceTests/TestProject.swift#L8

- Inside of `testProject`, all case of `Platform` are checked by for-loop

https://github.com/yonaskolb/XcodeGen/blob/49a9bda9925b4df5b87db912d84c10c418912bb8/Tests/PerformanceTests/TestProject.swift#L36

Reply:
Hint2:

`Platform.allCases` referes all case of platform so please consider to remove `visionOS`

Reply:
I decided to use a dummy value because some test codes would be complicated. Fixed: [e66fb5b](https://github.com/yonaskolb/XcodeGen/pull/1379/commits/e66fb5be7672b2c97c83d0956ecf5e522cab78a2)